### ISSUE TITLE
Command not found: fabricio

### ISSUE THREAD

**Issue by johndelong** [strongself/fabricio#53]
There is no `fabricio` command in the `/bin` directory in version 1.3.0. Please add another tag so that gem installation can use the `fabricio` command.

### Steps to Reproduce:
`> bundle init`
`> bundle add fabricio`
`> bundle exec fabricio`
`bundler: command not found: fabricio
Install missing gem executables with bundle install`

**Comment by fernandodev** [comment]
Yep, this isn't embedded there even in older versions. Please add the bin wrapper overthere 👍

**Comment by maydin** [comment]
Gem could be installed from source code using https://github.com/rdp/specific_install.

**Comment by spaghetti-** [comment]
For someone who is not familiar with Ruby, how do I add a bin wrapper? Wrapper over what?

**Comment by fernandodev** [comment]
@spaghetti-  Basically you would need to create a CLI for all commands in this gem.

**Comment by thyrlian** [comment]
The public gem `v1.3.0` doesn't work.

But if you build it locally, it's working:
```terminal
gem build fabricio.gemspec
gem install fabricio-1.3.0.gem
```

My guess is the gem you uploaded to rubygems.org was broken.  Could you please try to upload a new one?  Thanks.
